community/server/src/test/java/org/neo4j/server/integration/ServerConfigIT.java

```diff
@@ -49,23 +49,6 @@
     private CommunityNeoServer server;
 
     @Test
-    public void serverConfigShouldBeVisibleInJMX() throws Throwable
-    {
-        // Given
-        String configValue = tempDir.newFile().getAbsolutePath();
-        server = CommunityServerBuilder.server().withProperty(
-                ServerSettings.http_log_config_file.name(), configValue )
-                .build();
-
-        // When
-        server.start();
-
-        // Then
-        ObjectName name = getObjectName( server.getDatabase().getGraph(), ConfigurationBean.CONFIGURATION_MBEAN_NAME );
-        assertThat( getAttribute( name, ServerSettings.http_log_config_file.name() ), equalTo( (Object)configValue ) );
-    }
```

Comment:
This test isn't an HTTP logging test. You need to reinstate it with a different setting.